test/runnable/inline.d

```diff
@@ -780,6 +780,26 @@ void test7625()
 }
 
 /**********************************/
+// 9785 partial fix
+
+void test9785()
+{
+	int j = 3;
+
+        void loop(scope const void function(int x) dg) {
+	    pragma(inline, true);
+	    dg(++j);
```

Comment:
Nitpick: the indentation here seems to be off? Evil tab characters sneaking in again maybe?